[2025-02-18T04:31:06.044+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:31:06.059+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-27T00:00:00+00:00 [queued]>
[2025-02-18T04:31:06.066+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-27T00:00:00+00:00 [queued]>
[2025-02-18T04:31:06.066+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:31:06.078+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-03-27 00:00:00+00:00
[2025-02-18T04:31:06.088+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=358) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:31:06.092+0000] {standard_task_runner.py:72} INFO - Started process 361 to run task
[2025-02-18T04:31:06.089+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-03-27T00:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpiu8e77lr']
[2025-02-18T04:31:06.093+0000] {standard_task_runner.py:105} INFO - Job 40: Subtask hello_task
[2025-02-18T04:31:06.121+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:06.166+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-03-27T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:31:06.168+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:06.330+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-27T00:00:00+00:00'
[2025-02-18T04:31:06.332+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:31:06.356+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:31:06.360+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:31:06.371+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:31:06.372+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-03-27T00:00:00+00:00, execution_date=20240327T000000, start_date=20250218T043106, end_date=20250218T043106
[2025-02-18T04:31:06.490+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:31:06.505+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-18T16:34:08.950+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T16:34:08.993+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-27T00:00:00+00:00 [queued]>
[2025-02-18T16:34:09.012+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-27T00:00:00+00:00 [queued]>
[2025-02-18T16:34:09.021+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T16:34:09.075+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-03-27 00:00:00+00:00
[2025-02-18T16:34:09.096+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-03-27T00:00:00+00:00', '--job-id', '409', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmp15x1m1x8']
[2025-02-18T16:34:09.101+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1640) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T16:34:09.115+0000] {standard_task_runner.py:72} INFO - Started process 1649 to run task
[2025-02-18T16:34:09.109+0000] {standard_task_runner.py:105} INFO - Job 409: Subtask hello_task
[2025-02-18T16:34:09.181+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:34:09.280+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-03-27T00:00:00+00:00 [running]> on host 8720d9c228eb
[2025-02-18T16:34:09.288+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:34:09.565+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-27T00:00:00+00:00'
[2025-02-18T16:34:09.569+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T16:34:09.597+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T16:34:09.600+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T16:34:09.628+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T16:34:09.646+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-03-27T00:00:00+00:00, execution_date=20240327T000000, start_date=20250218T163408, end_date=20250218T163409
[2025-02-18T16:34:09.915+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T16:34:09.961+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T16:34:09.966+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
